src/main/java/frc/robot/Constants.java

```diff
@@ -145,19 +154,52 @@ public static class TurretConstants {
         TURRET_GEAR_TEETH
             / (LOW_GEAR_CANCODER_TEETH
                 + 0.0); // Revolutions of the turret, to revolutions of the cancoder
-    public static final int highGearCAN_CODER_ID = 0;
-    public static final int lowGearCAN_CODER_ID = 0;
+    public static final int highGearCAN_CODER_ID = 5;
+    public static final int lowGearCAN_CODER_ID = 6;
     public static final double ENCODER_ZERO_OFFSET_FROM_TURRET_ZERO_REV = 0;
 
     // Turret limits
     public static final double TURRET_MIN_ANGLE_ROTATIONS =
         (TURRET_MIN_ANGLE_DEGREES / (360 * TURRET_MOTOR_TURRET_RATIO));
     public static final double TURRET_MAX_ANGLE_ROTATIONS =
-        (TURRET_MAX_ANGLE_DEGREES / (360 * TURRET_CONVERSION_FACTOR));
+        (TURRET_MAX_ANGLE_DEGREES / (360 * TURRET_MOTOR_TURRET_RATIO));
     public static final double PITCH_MIN_ANGLE_ROTATIONS =
-        (PITCH_MIN_ANGLE_DEGREES / (360 * PITCH_CONVERSION_FACTOR));
+        ((PITCH_MIN_ANGLE_DEGREES * PITCH_CONVERSION_FACTOR) / 360);
     public static final double PITCH_MAX_ANGLE_ROTATIONS =
-        (PITCH_MAX_ANGLE_DEGREES / (360 * PITCH_CONVERSION_FACTOR));
+        ((PITCH_MAX_ANGLE_DEGREES * PITCH_CONVERSION_FACTOR) / 360);
+    public static final Rotation2d PIN_EPSILION = Rotation2d.fromDegrees(2);
+
+    public static final TurretDataPoint[] TURRET_DATA = {
```

Comment:
Let's clean this up.  Either inches, or meters, and put the data points in order.  That way we can see at a glance where we have data points